packages/flutter_tools/lib/src/version.dart

```diff
@@ -26,7 +27,7 @@ enum Channel {
 
 // Beware: Keep order in accordance with stability
 const Set<String> kOfficialChannels = <String>{
-  'master',
+  globals.kDefaultFrameworkChannel,
```

Comment:
when we do the migration we'll want to move people on `master` to `main` automatically, it won't just be a matter of renaming a constant.

Reply:
acknowledged. i'm working on a design doc for this.

Reply:
I just figured while I was at it.